django/econsensus/publicweb/urls.py

```diff
@@ -8,7 +8,7 @@
                     FeedbackSnippetCreate, FeedbackUpdate, \
                     EconsensusActionitemCreateView, EconsensusActionitemUpdateView, \
                     EconsensusActionitemListView, OrganizationRedirectView, \
-                    YourDetails, \
+                    YourDetails, ChangeOwnerView, \
```

Comment:
You're quite reasonably following bad habits that have gone before, but each of these imports should be on a separate line. Makes future diffing much easier to read.